### DISCUSSION TITLE
Aiko — Free native app for macOS and iOS

### DISCUSSION THREAD

**Discussion by sindresorhus** [ggml-org/whisper.cpp#849] [Show and tell]
https://sindresorhus.com/aiko

Aiko lets you run Whisper locally on your Mac, iPhone, and iPad.

Highlights:
- Reader and timestamp view
- Record audio
- Export to text, JSON, CSV, subtitles
- Shortcuts support

The app uses the Whisper large v2 model on macOS and the medium or small model on iOS depending on available memory.

I'm planning to take advantage of the CoreML support soon, which will increase the performance on iOS significantly.

I'm actively working on more features and improvements. However, I intend to keep the app simple and read-only. For editing capabilities and more advanced needs, check out [MacWhisper](https://goodsnooze.gumroad.com/l/macwhisper).

And huge thanks to @ggerganov for making it possible to make this app 🙏

![](https://sindresorhus.com/_astro/screenshot1.2ecc32f3.jpg)
![iOS](https://user-images.githubusercontent.com/170270/235439998-b237ce86-bfc2-4fca-9a9d-692830dd180c.png)

**Comment by BuildBackBuehler** [comment]
Heya, thanks so much for releasing this and keeping it free (= (ofc I hope you're handsomely rewarded for your efforts by devs...or otherwise!). 

I'm interested in using this for live transcription on my iPad (M1 8GB VRAM) and streaming it to my Mac or my NAS (to then pump into Pipelines, acts as an OpenAI API-based server/middleman, and use that text for commands/chat messages @ Aider or my LLM/Open-WebUI), is that possible? I imagine it might be...but even if it is, probably better off just taking the 2GB VRAM hit on my Mac(?) Or at least that's what it seems like it'd take based on the ~750M Params. for Whisper-Large-V3_turbo. Thoughts? Thanks!!
